lib/cqm/converter/qdm_patient.rb

```diff
@@ -223,13 +226,13 @@ def unpack_components(hds_attrs)
 
     # Unpack diagnosis.
     def unpack_diagnosis(hds_attrs)
-      if hds_attrs.key?('diagnosis') && !hds_attrs['diagnosis'].nil?
+      if hds_attrs.key?('diagnosis') && !hds_attrs['diagnosis'].empty?
```

Comment:
What was the issue being hit before this change, if I may ask?

Reply:
I suspect what was happening here was that `hds_attrs['diagnosis']` was not `nil`, but didn't contain any information; either it was an empty `Hash` or an empty `Array`.

Reply:
Right